Dataset_Sheet

```diff
@@ -0,0 +1 @@
+
```

Comment:
![download](https://github.com/user-attachments/assets/93cb1a8a-93ee-431b-ac55-427ef3e0345a)
This image was made by adding noise to make it look more gritty and blurry. These colors appear on the image because the original photo has been photoshopped with vibrant colors on the vine to make the vines look unnatural but stand out from everything else. From the right column of the photos mostly magenta was used, in the middle column mostly blue was used and in the left column mostly orange was used. The middle row uses more vibrant color compared to the top row, while the bottom row uses less color. 
![image](https://github.com/user-attachments/assets/0ae85b0e-51fe-46cb-999e-73b696041f13)
This was another image I made by adding noise to it. This was the engineering building with photoshopped blue vines growing on the side of the building